pkg/plugins/autodiscovery/dockercompose/utils.go

```diff
@@ -0,0 +1,85 @@
+package dockercompose
+
+import (
+	"fmt"
+	"io"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/sirupsen/logrus"
+	goyaml "gopkg.in/yaml.v3"
+)
+
+// searchDockerComposeFiles will look, recursively, for every files named Chart.yaml from a root directory.
+func searchDockerComposeFiles(rootDir string, files []string) ([]string, error) {
```

Comment:
```suggestion
func searchDockerComposeFiles(rootDir string, filePatterns []string) ([]string, error) {
```

Using `files` as a name is confusing (is it a file found by the `filepath.Walk()` ? )